core/lib/libmbedtls/include/mbedtls_hash.h

```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
I don't see the point with this file.

Reply:
As above, I will move the needed included header into `tee_lmd_provider.c` directly.